TKGame/Character.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame
+{
+    /*internal*/ class Character
+    {
+        Texture2D texture;
+        Vector2 pos;
+        Vector2 vel;
+        bool jumped;
+
+        // Character constructor
+        public Character(Texture2D newTexture, Vector2 newPos)
+        {
+            texture = newTexture;
+            pos = newPos;
+            jumped = true;
+        }
+
+        // Player character movement, gravity, jumping 
+        public void Update(GameTime gameTime)
+        {
+            pos += vel;
+
+            // Move to right
+            if (Keyboard.GetState().IsKeyDown(Keys.Right))
+                vel.X = 3f;
+            // Move to left
+            else if (Keyboard.GetState().IsKeyDown(Keys.Left))
+                vel.X = -3f;
+            // No key pressed so don't move
+            else
+                vel.X = 0f;
+
+            // No double jumps due to hasJumped boolean
+            if (Keyboard.GetState().IsKeyDown(Keys.Space) && jumped == false)
```

Comment:
Why do we need to wait until the play jumps? The player will be use collisions to stop from falling into the floor so we should code gravity to be constant acceleration like real life.